h2o-py/h2o/backend/server.py

```diff
@@ -61,7 +61,7 @@ class H2OLocalServer(object):
 
     @staticmethod
     def start(jar_path=None, nthreads=-1, enable_assertions=True, max_mem_size=None, min_mem_size=None,
-              ice_root=None, port="54321+", verbose=True):
+              ice_root=None, port="54321+", extra_classpath=None, verbose=True):
```

Comment:
The plus sign seems to be mandatory:

                if not(port[-1] == "+" and port[:-1].isdigit()):
                    raise H2OValueError("`port` should be of the form 'DDDD+', where D is a digit. Got: %s" % port)

I guess it is supposed to inform the user that this is a base port not the actual port that will be used.